test/distributed/test_ddp_under_dist_autograd.py

```diff
@@ -1,596 +1,7 @@
 #!/usr/bin/env python3
```

Comment:
I think the `run_tests` import needs to be added back to this file to fix the test failures?